Library/Services/INfieldWavesService.cs

```diff
@@ -0,0 +1,34 @@
+﻿//    This file is part of Nfield.SDK.
+//
+//    Nfield.SDK is free software: you can redistribute it and/or modify
+//    it under the terms of the GNU Lesser General Public License as published by
+//    the Free Software Foundation, either version 3 of the License, or
+//    (at your option) any later version.
+//
+//    Nfield.SDK is distributed in the hope that it will be useful,
+//    but WITHOUT ANY WARRANTY; without even the implied warranty of
+//    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//    GNU Lesser General Public License for more details.
+//
+//    You should have received a copy of the GNU Lesser General Public License
+//    along with Nfield.SDK.  If not, see <http://www.gnu.org/licenses/>.
+
+using System.Linq;
+using System.Threading.Tasks;
+using Nfield.Models;
+
+namespace Nfield.Services
+{
+    public interface INfieldWavesService
```

Comment:
I understand your suggestion because a Wave survey belongs to a ParentSurvey nevertheless:
Because in the manager repo we refer in the code (controllers/handlers etc) to ParentSurveys and Waves or WaveSurveys I think we should keep the format in all other repositories for consistency.

Reply:
I don't agree to leave it in this way (at least in the SDK) because now we don't have  consistence between the routing, the SDK services and the code.
We need to take into account the customers, it is assumed that we are following REST API conventions `object/{if}/object/{id}` with a structure parent->children. 
We should follow the contracts from the API conventions and not in the opposite side, so the inconsistency is in glue not in the public API. 

I am not telling you to change Glu, but we don't want to move to the client side the inconsistencies. We can leave in the public API the class names name as it is now because most  important is the routing. But the SDK is also public, there we need to maintain the consistency with the routing.

Many old controllers do not follow rightly the REST API conventions and is because we are going to create a V2, but need to take care with the new controllers.

Reply:
You made a very good point. 
I applied the changes for the sdk. Customer first always! Did the changes 🥇